tools/loader/Loader.cpp

```diff
@@ -31,6 +31,13 @@
 
 using namespace glow;
 
+/// -enable-rowwise - Command line option to enable ENABLE_ROWWISE statements.
```

Comment:
can you expand here `Command line option to enable ENABLE_ROWWISE statements`
with more details. `ENABLE_ROWWISE statements` does not tell much.